modules/ROOT/pages/download-and-install-service-mesh.adoc

```diff
@@ -81,5 +81,27 @@ Anypoint Service Mesh is installed and the status of all pods is displayed:
 +
 image::verified-service-mesh-installer-output.png[]
 
+=== Configure Headers for Client Authorization-based Policies
 
+Anypoint Service Mesh is configured with minimized whitelisted headers for client-based policies:
```

Comment:
Are these headers that have been "whitelisted", or "whitelist" (type) headers?

Reply:
there are no whitelist type headers. Whitelisting means "approving" or adding to an approved list.